examples/keras_track.py

```diff
@@ -1,6 +1,6 @@
 from aim.keras import AimCallback
 
-import keras
+from tensorflow import keras
```

Comment:
This line should remain as `import keras`, because before tf2 keras was a separate high-level ML framework.

Reply:
👌